src/main/features/ccj/index.ts

```diff
@@ -27,11 +27,16 @@ export class CCJFeature {
     app.all(allCCJ, requestHandler())
     app.all(allCCJ, ClaimMiddleware.retrieveByExternalId)
     app.all(allCCJ, OnlyClaimantLinkedToClaimCanDoIt.check())
-    app.all(/^\/case\/.+\/ccj\/(?!confirmation).*$/, CCJGuard.requestHandler)
-    app.all(/^\/case\/.+\/ccj\/(?!confirmation).*$/,
+    app.all(/^\/case\/[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}\/ccj\/(?!confirmation).*$/, CCJGuard.requestHandler)
+    app.all(/^\/case\/[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}\/ccj\/(?!confirmation).*$/,
```

Comment:
instead of adding this regex here could we do `county-court-judgment` as the path for the new `ccj` journey in claimant-response?

Reply:
Done